src/providers/collection.js

```diff
@@ -6,12 +6,12 @@ exports.getTable = table => this.database.get(table) || this.database.set(table,
 
 exports.getAll = (table) => {
```

Comment:
This should be changed to
```js
exports.getAll(table) => {
  return Array.from(this.getTable(table).values());
}
```

getTable will automatically create the table for you (as it should), which means there will never be a time when the table doesn't exist and therefore no reason to check for the collection beforehand or return an empty array/null

Reply:
```js
exports.getAll(table) => Array.from(this.getTable(table).values());
```
actually this^